mephisto/abstractions/blueprint.py

```diff
@@ -262,8 +264,47 @@ def get_frontend_args(self) -> Dict[str, Any]:
         Specifies what options should be fowarded
         to the client for use by the task's frontend
         """
+        self.update_task_config_with_tips(self.frontend_task_config)
         return self.frontend_task_config.copy()
 
+    def get_tips_directory(self) -> Optional[str]:
+        """
+        Returns the tips directory as set in hydra config
+        By default this is '${task_dir}/outputs/tips'
+        """
+        path_to_tips = self.args["blueprint"]["tips_location"]
```

Comment:
`self.args.blueprint.tips_location` is the better path to access here (treating as a `DictConfig`) rather than a `Dict`